flex/03-flex-header-2/solution/solution.html

```diff
@@ -19,6 +19,7 @@
           <li><a href="google.com">link-three</a></li>
         </ul>
       </div>
+    </div>
```

Comment:
issue (blocking): This adds an extraneous `</div>`

The original HTML looks correct as the `left` and `right` `<div>` containers are both intended to nest under the `header`